cni/network/invoker_cns.go

```diff
@@ -450,6 +455,13 @@ func configureSecondaryAddResult(info *IPResultInfo, addResult *IPAMAddResult, p
 		return errors.Wrap(err, "Invalid mac address")
 	}
 
+	_, hostIPNet, err := net.ParseCIDR(info.hostSubnet)
```

Comment:
why this is required?

Reply:
because we do need "hostSubnetPrefix" to find master interface if you check findMasterInterface()